lib/openstax/aws/build_image_command_1.rb

```diff
@@ -50,6 +54,9 @@ def to_s
         @packer.to_s
       end
 
+      def status 
+        @packer.debug
+      end
```

Comment:
I'm little confused how this is supposed to work. After installing Packer, when i added `command.status` after it finished running it gave 
```
Traceback (most recent call last):
	2: from ./build_image:55:in `<main>'
	1: from ./build_image:47:in `main'
/...../aws-ruby/lib/openstax/aws/build_image_command_1.rb:63:in `status': undefined method `debug' for #<OpenStax::Aws::Packer_1_4_1:0x000055a9074f7da0> (NoMethodError)
Did you mean?  debug!
	3: from ./build_image:55:in `<main>'
	2: from ./build_image:45:in `main'
	1: from ./build_image:51:in `rescue in main'
/....../aws-ruby/lib/openstax/aws/build_image_command_1.rb:63:in `status': undefined method `debug' for #<OpenStax::Aws::Packer_1_4_1:0x000055a9074f7da0> (NoMethodError)
```
Looking at the `@debug` (which can be changed from false to true by `@debug!`) in packer classes it seems to be used to control the mode of command that is going to be executed rather than storing any error messages.

Running the command with `--verbose` shows the desired error even when running the script in `apps/highlights` so maybe they just need to be filtered out in
```
Open3.popen2e(command) do |stdin, stdout_err, wait_thr|
  while line=stdout_err.gets do
    puts(line) if line =~ / amazon-ebs\: /
  end
end
```
based on some additional flag if only the errors are meant to be shown